machine_translation/__init__.py

```diff
@@ -520,6 +519,7 @@ def main(config, tr_stream, dev_stream, bokeh=False):
 
     # Plot cost in bokeh if necessary
     if bokeh:
+        from blocks.extensions.plot import Plot
```

Comment:
i'm not sure about the Blocks convention for imports in function bodies or code blocks, for me i do not prefer them because of dependency tracking, but i guess we can ask @dmitriy-serdyuk